package.json

```diff
@@ -7,9 +7,13 @@
     "test:watch": "jest --watch",
     "test": "jest -c jest.config.js",
     "coverage": "jest -c jest.config.js --collect-coverage",
+    "eslint": "eslint --config .eslintrc.js \"**/*.{ts,js}\"",
```

Comment:
We don't have an `.ts` in Satellite, so this can be `.js` only.

Reply:
Done!